<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>help</class>
 <widget class="QWidget" name="help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>1115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mfix_local_label">
     <property name="text">
      <string>&lt;a href=&quot;%s&quot;&gt;Local MFiX Documentation (v %s)&lt;/a&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mfix_online_label">
     <property name="text">
      <string>&lt;a href=&quot;https://mfix.netl.doe.gov/doc/mfix/latest&quot;&gt;Online MFiX Documentation (latest version)&lt;/a&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="nodeworks_local_label">
     <property name="text">
      <string>&lt;a href=&quot;%s&quot;&gt;Local Nodeworks Documentation (v %s)&lt;/a&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="nodeworks_online_label">
     <property name="text">
      <string>&lt;a href=&quot;https://mfix.netl.doe.gov/doc/nodeworks/latest&quot;&gt;Online Nodeworks Documentation (latest version)&lt;/a&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QWidget#help{background-color: white;}</string>
     </property>
     <property name="text">
      <string>&lt;b&gt;Tutorials&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QListWidget" name="tutorial_listwidget">
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
